### ISSUE TITLE
Pins disappearing 

### ISSUE THREAD

**Issue by Qct00** [MehVahdJukaar/mapatlases-neoforge#160]
people keep complaining on my server, that every time they log off and log back on their pins disappear.

**Comment by MehVahdJukaar** [comment]
can you try and see if it happens on a server with just the mod on?

**Comment by MehVahdJukaar** [comment]
I cant replicate. I tested on fabric version on a server. 
Can you verify the mod is saving stuf in the map_atlases folder?
look in your instance directory /map_atlases/multiplayer folder. there should be some files named with the ip of the server

**Comment by Klfksdgdflkg** [comment]
Hi. I'm having this problem on my server and checked and it is not adding any files to the multiplayer folder. What can I do to fix this? I appreciate the help!

**Comment by Lebleus** [comment]
I can confirm. The folder didn't create itself.

**Comment by MaxWasUnavailable** [comment]
@MehVahdJukaar So, all of our pins are getting removed. I did some cursory reading over your code, and I can only assume something is going wrong here: https://github.com/MehVahdJukaar/mapatlases-neoforge/blob/96e1b6d1777ff74b5c44cc3a127794bb911c92e4/common/src/main/java/pepjebs/mapatlases/integration/moonlight/ClientMarkers.java#L108 (that is, assuming it is being called at all, which an extra debug log would be very helpful with).

I'm not getting an error logged related to markers not being saved. It might be good to add a separate case here when the path is null, and log a warning? I don't see a case where this should happen, so it'd be good to log.

Is there any way to verify the load call went through correctly? And are markers stored client-side, or server-side? Where should I be able to find them (if they were actually saving)? That'd help me pinpoint whether they're being saved, and not loaded, or whether they're not being saved at all.

**Comment by MehVahdJukaar** [comment]
I have som experimental changes here. Please try it out and verify it creates a folder with a name that should match the sever adress in map_atlases/multiplayer
[map_atlases-1.20-6.0.11.zip](https://github.com/user-attachments/files/18311163/map_atlases-1.20-6.0.11.zip)
[map_atlases-1.20-6.0.11-fabric.zip](https://github.com/user-attachments/files/18311164/map_atlases-1.20-6.0.11-fabric.zip)

Rename .zip to .jar because GH wont let me upload that here

**Comment by MaxWasUnavailable** [comment]
Will do! Is this on a branch so I can check it out? :)

**Comment by MehVahdJukaar** [comment]
yeah

**Comment by MaxWasUnavailable** [comment]
@MehVahdJukaar Seems to have fixed it. Thanks for the quick response!

**Comment by MehVahdJukaar** [comment]
Finally nice

**Comment by rafaelkai** [comment]
Still happening for me on latest version in 1.20.1, nothing is created on the multiplayer folder

**Comment by MehVahdJukaar** [comment]
yeah but comments above says that it works for them and it does work for me too.
You myst be usind an old version or its a mod incompatibility
